docs/_articles/en/apex/isv-debugger.md

```diff
@@ -11,6 +11,7 @@ An ISV can reproduce issues in the specific environment, so problems can be diag
 
 ISV Customer Debugger is part of the Apex Interactive Debugger (`salesforcedx-vscode-apex-debugger`) extension, so you don’t need to install anything other than the Salesforce Extension Pack and its prerequisites. You can debug only sandbox orgs.
 
+1. Log into your License Management Org (LMO) and enable the **View and Debug Managed Apex** permission for your user via Profiles or Permission Sets.
```

Comment:
```suggestion
1. Log into your License Management Org (LMO) and enable the **View and Debug Managed Apex** permission for your user in Setup using Profiles or Permission Sets.
```

Reply:
Assuming Setup and potentially multiple users. Also, assuming most dev don't have access to the LMO. So, should we say?
Contact your the License Management Org (LMO) admin to enable the View and Debug Managed Apex permission for users. (Admins will know whether they need to do this via a profile or perm set - I think we can safely assume that.)